assignment6/rearranging_cars.py

```diff
@@ -0,0 +1,72 @@
+
+class Movement(object):
+    """This class stores information about one movement,
+    i.e. a car moving from position *fro* to position *to*.
+    """
+
+    def __init__(self, car, fro, to):
+        self.car = car
+        self.fro = fro
+        self.to = to
+
+    def print_movement(self):
+        print("Move car %s from %s to %s" %(self.car, self.fro, self.to))
+
+def rearrange(current_permutation, final_permutation):
+    """Rearrange the cars so that their positions are in accordance to
+    final_permutation.
+
+    Args:
+        current_permutation: the initial car permutation in the parking lot.
+        final_permutation: the desired permutation.
+
+    Returns:
+        movements: a list of Movement objects. Represents the consecutive
+                   movements needed to rearrange the cars from current_permutation
+                   to final_permutation.
+    """
+    car_to_pos = dict(zip(current_permutation,
```

Comment:
There are neater way to create this dictionary, for example like @muzaluisa did, if you want to give a look at her code!
I like that you use this data structure :)